### ISSUE TITLE
tox isort failing

### ISSUE THREAD

**Issue by AndreMiras** [AndreMiras/pyetheroll#5]
`isort` is failing when running from `tox` in local, but not in Travis.
The command ran was `tox` and the logs:
```
isort runtests: commands[1] | pipenv run isort --check-only --recursive --diff pyetheroll/ tests/ setup.py setup_meta.py
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
ERROR: /home/andre/workspace/pyetheroll/pyetheroll/etherscan_utils.py Imports are incorrectly sorted.
--- /home/andre/workspace/pyetheroll/pyetheroll/etherscan_utils.py:before       2019-03-21 22:22:39.487752
+++ /home/andre/workspace/pyetheroll/pyetheroll/etherscan_utils.py:after        2019-03-22 22:36:15.582289
@@ -4,7 +4,6 @@

 from etherscan.accounts import Account as EtherscanAccount
 from etherscan.contracts import Contract as EtherscanContract
-
 from pyetheroll.constants import ChainID

 logger = logging.getLogger(__name__)
ERROR: /home/andre/workspace/pyetheroll/pyetheroll/transaction_debugger.py Imports are incorrectly sorted.
--- /home/andre/workspace/pyetheroll/pyetheroll/transaction_debugger.py:before  2019-03-21 22:22:39.487752
+++ /home/andre/workspace/pyetheroll/pyetheroll/transaction_debugger.py:after   2019-03-22 22:36:15.587898
@@ -2,11 +2,10 @@

 from eth_abi import decode_abi
 from eth_utils import decode_hex, function_abi_to_4byte_selector
-from web3 import HTTPProvider, Web3
-
 from pyetheroll.constants import ChainID
 from pyetheroll.etherscan_utils import (ChainEtherscanContractFactory,
                                         get_etherscan_api_key)
+from web3 import HTTPProvider, Web3


 def decode_contract_call(contract_abi: list, call_data: str):
ERROR: /home/andre/workspace/pyetheroll/pyetheroll/etheroll.py Imports are incorrectly sorted.
--- /home/andre/workspace/pyetheroll/pyetheroll/etheroll.py:before      2019-03-21 22:22:39.487752
+++ /home/andre/workspace/pyetheroll/pyetheroll/etheroll.py:after       2019-03-22 22:36:15.602631
@@ -11,16 +11,15 @@
 from eth_utils import to_checksum_address
 from etherscan.client import EmptyResponse
 from hexbytes.main import HexBytes
-from web3 import Web3
-from web3.auto import w3
-from web3.contract import Contract
-
 from pyetheroll.constants import DEFAULT_GAS_PRICE_GWEI, ROUND_DIGITS, ChainID
 from pyetheroll.etherscan_utils import (ChainEtherscanAccountFactory,
                                         ChainEtherscanContractFactory,
                                         get_etherscan_api_key)
 from pyetheroll.transaction_debugger import (HTTPProviderFactory,
                                              TransactionDebugger)
+from web3 import Web3
+from web3.auto import w3
+from web3.contract import Contract

 REQUESTS_CACHE_PARAMS = {
     'cache_name': 'requests_cache',
ERROR: /home/andre/workspace/pyetheroll/tests/test_transaction_debugger.py Imports are incorrectly sorted.
--- /home/andre/workspace/pyetheroll/tests/test_transaction_debugger.py:before  2019-03-21 20:13:25.602622
+++ /home/andre/workspace/pyetheroll/tests/test_transaction_debugger.py:after   2019-03-22 22:36:15.615821
@@ -3,11 +3,10 @@
 from unittest import mock

 from hexbytes.main import HexBytes
-from web3.datastructures import AttributeDict
-
 from pyetheroll.constants import ChainID
 from pyetheroll.transaction_debugger import (TransactionDebugger,
                                              decode_contract_call)
+from web3.datastructures import AttributeDict


 class TestTransactionDebugger(unittest.TestCase):
ERROR: /home/andre/workspace/pyetheroll/tests/test_etheroll.py Imports are incorrectly sorted.
--- /home/andre/workspace/pyetheroll/tests/test_etheroll.py:before      2019-03-21 20:13:25.602622
+++ /home/andre/workspace/pyetheroll/tests/test_etheroll.py:after       2019-03-22 22:36:15.633403
@@ -10,7 +10,6 @@
 import eth_account
 from eth_account.internal.transactions import assert_valid_fields
 from eth_keyfile import create_keyfile_json
-
 from pyetheroll.constants import ChainID
 from pyetheroll.etheroll import Etheroll

ERROR: InvocationError: '/home/andre/workspace/pyetheroll/.tox/isort/bin/pipenv run isort --check-only --recursive --diff pyetheroll/ tests/ setup.py setup_meta.py'
```

**Comment by AndreMiras** [comment]
Looks like the regression happened in this commit 2ac13b1ce1503995585fa58887f881c068c01537

**Comment by AndreMiras** [comment]
Closed in 891fdaac556a14d090711ff2de21f6c18615893c
